ansys/mapdl/core/commands.py

```diff
@@ -211,4 +211,34 @@ class Commands(
 
     """Wrapped MAPDL commands"""
 
-    pass
+
+class CommandOutput(str):
+    """Customized Command Output class"""
+
+    ## References:
+    # - https://stackoverflow.com/questions/7255655/how-to-subclass-str-in-python
+    # - https://docs.python.org/3/library/collections.html#userstring-objects
+    # - Source code of UserString
+
+    def __new__(cls, content, cmd=None):
+        obj = super().__new__(cls, content)
+        obj._cmd = cmd
+        return obj
+
+    @property
+    def cmd(self):
+        return self._cmd.split(',')[0]
+
+    @cmd.setter
+    def cmd(self, cmd):
+        """Not allowed to change the value of ``cmd``."""
+        pass
```

Comment:
Best not to have things silently fail.
```suggestion
        raise AttributeError("The `cmd` attribute cannot be set")
```

Reply:
Very good idea. Thank you.